kart/point_cloud/import_.py

```diff
@@ -258,16 +176,15 @@ def _format_array(array):
 
 
 def get_copc_version(info):
-    vlr_0 = info.get("vlr_0")
-    if vlr_0:
-        user_id = vlr_0.get("user_id")
-        if user_id == "copc":
-            return vlr_0.get("record_id")
-    return NOT_COPC
+    # TODO - find out how to extract the COPC version number using the current PDAL.
```

Comment:
https://copc.io/

The first (E)VLR in the file (COPC info VLR) should have UserID=`copc` & RecordID of `1` (it's a little endian unsigned short : 2 bytes)

Which should be the same as:
*  verify that the bytes at offsets 393 and 394 contain the values 1 and 0, respectively (this is the COPC version number, 1).

Reply:
Not done - will do in a separate change. Returning "1.0" for COPC will do for now (especially since there is only 1 COPC version right now)

I was previously using PDAL to extract this VLR, but that stopped working, but now PDAL is exposing whether or not a PC tile is COPC at a high level. I need to get a hex editor and dig into why the low level way stopped working with a hex editor - is PDAL hiding VLRs from me if it has already handled them at a lower level? Or have I made some mistake somewhere

Reply:
PDAL now hides the COPC VLRs from me - see
https://github.com/PDAL/PDAL/blob/3e33800d85d48f726dcd0931cefe062c4af2b573/io/private/las/Vlr.cpp#L53
But does expose whether or not a tile "is COPC" - as the code now checks. Just not the COPC version number (which may one day be useful, but is low priority right now)

Reply:
I think because COPC VLRs are content-dependent in general it's bad for them to flow through PDAL pipelines and that's been disabled. Not sure whether there's another option for querying.

Yeah, is low priority